### ISSUE TITLE
Podspec requires iOS7 min, SDK crashes on iOS7

### ISSUE THREAD

**Issue by zrxq** [googlevr/gvr-ios-sdk#68]
Podspec [contains](https://github.com/CocoaPods/Specs/blob/master/Specs/GVRSDK/0.8.1/GVRSDK.podspec.json#L13)
```
"platforms": {
    "ios": "7.0"
  }
```

But when trying to invoke `[GVRVideoView init]` on an iOS7 device, the app crashes with
```
+[UIImage imageNamed:inBundle:compatibleWithTraitCollection:]: unrecognized selector sent to class 0x3ab6baf0
```

**Comment by yalong5837** [comment]
+1

**Comment by nathanmartz** [comment]
Yes. This is an error and will be fixed in our next update to raise the min to iOS 8.

**Comment by dav-cz** [comment]
Fixed in v0.9.0
https://developers.google.com/vr/ios/release-notes
